Examples/MAX78000/CNN/asl_demo/main.c

```diff
@@ -468,14 +468,15 @@ uint8_t check_inference(q15_t *ml_soft, int32_t *ml_data, int16_t *out_class, do
             *out_prob = 100.0 * max / 32768.0;
             /// MXC_TFT_ClearScreen();
             memset(buff, 32, TFT_BUFF_SIZE);
-            sprintf(buff, "%s (%0.1f%%)", classes[max_index], (double)100.0 * max / 32768.0);
+            snprintf(buff, sizeof(buff), "%s (%0.1f%%)", classes[max_index],
+                     (double)100.0 * max / 32768.0);
             TFT_Print(buff, 100, 8, font_1);
-            //sprintf(buff, "__________________________ ");
+            //snprintf(buff, sizeof(buff), "__________________________ ");
             //TFT_Print(buff, 1, 50, urw_gothic_12_white_bg_grey);
-            //sprintf(buff, "Top classes:");
+            //snprintf(buff, sizeof(buff), "Top classes:");
             //TFT_Print(buff, 1, 80, urw_gothic_13_white_bg_grey);
         } else {
-            //sprintf(buff, "%s (%0.1f%%)", classes[max_index],
+            //snprintf(buff, sizeof(buff), "%s (%0.1f%%)", classes[max_index],
             //         (double) 100.0 * max / 32768.0);
             //TFT_Print(buff, 20, 80 + 20 * top, urw_gothic_12_white_bg_grey);
```

Comment:
Is the commented code something that's still in development and requires more testing? Or should it be deleted? This else block does nothing.

Reply:
It's ok to delete commented lines